plugin/backend_test.go

```diff
@@ -14,12 +16,14 @@ const (
 	maxLeaseTTLHr     = 12
 )
 
-func getTestBackend(tb testing.TB) (logical.Backend, logical.Storage) {
+func getTestBackend(tb testing.TB) (*backend, logical.Storage) {
 	tb.Helper()
 
 	config := logical.TestBackendConfig()
 	config.StorageView = new(logical.InmemStorage)
-	config.Logger = hclog.NewNullLogger()
+	if os.Getenv("VAULT_LOG") == "" {
```

Comment:
What does this do? I'm assuming if `config.Logger == nil` other stuff might have issues?

Reply:
Made changes in f16c3d6ec7dbe88b00eacbb6320a0675743b2363